hip/matrix/csr_kernels.hip.cpp

```diff
@@ -386,7 +386,7 @@ void advanced_spmv(std::shared_ptr<const HipExecutor> exec,
             GKO_NOT_SUPPORTED(nwarps);
         }
     } else if (a->get_strategy()->get_name() == "sparselib" ||
-               a->get_strategy()->get_name() == "cusparse") {
+               a->get_strategy()->get_name() == "hipsparse") {
```

Comment:
It should be "cusparse"  to keep the "cusparse" working after changing executor although it is changed by script.

Reply:
This was purely a search/replace issue. This change should not have been in the PR.